src/pages/getting-started/app-guides/in-app-upgrades/index.mdx

```diff
@@ -98,7 +98,7 @@ Before you can publish your upgrade, it needs to successfully go through the rev
 
 ## Upgrade activation
 
-Users can activate upgrades from the Marketplace or in **Settings > Integrations & apps** in the Agent App.
+Users can activate upgrades from the Marketplace or in **Settings > Integrations & apps** in the LiveChat App.
```

Comment:
ten cały BB nie jest jeszcze postanowione czy będzie w HD czy nie, jest szansa, że nie dlatego poprawiłam jedynie to póki co

Reply:
nie będzie póki co